src/components/CommentForm/CommentForm.tsx

```diff
@@ -13,10 +14,12 @@ import s from './CommentForm.module.css';
 
 interface CommentFormProps {
     actionButton: React.ReactNode;
+    childrenProps?: React.ReactNode;
```

Comment:
Looks like you dont need this prop, you can move RejectInterviewStatus into this component